CONTRIBUTING.md

```diff
@@ -95,9 +95,9 @@ why we ask this as well as instructions for how to proceed, see the
   sudo yum update -y
   sudo yum groupinstall -y 'Development Tools'
   sudo yum install -y postgresql11-devel postgresql11-server     \
-                      libxml2-devel libxslt-devel openssl-devel  \
-                      pam-devel readline-devel git libcurl-devel \
-                      llvm5.0 llvm-toolset-7-clang
+                      git libcurl-devel libxml2-devel libxslt-devel \
+                      libzstd-devel llvm-toolset-7-clang llvm5.0 lz4 lz4-devel \
```

Comment:
do we need lz4? I guess lz4-devel should be enough

Reply:
I added `lz4` to be consistent with https://github.com/citusdata/the-process/pull/49/ However it seems that it is not necessary.

lz4-devel depends on lz4. See 
https://centos.pkgs.org/8/centos-baseos-x86_64/lz4-devel-1.8.3-2.el8.i686.rpm.html